static/app/views/monitors/components/monitorForm.tsx

```diff
@@ -365,7 +365,9 @@ function MonitorForm({
                       stacked
                       inline={false}
                     />
-                    {parsedSchedule && <CronstrueText>"{parsedSchedule}"</CronstrueText>}
+                    {parsedSchedule && (
+                      <CronstrueText>&quot;{parsedSchedule}&quot;</CronstrueText>
```

Comment:
I think this case is where the lint rule really helps out. Before the `"` was hard to see inside all the other syntax and special chars... but now that we write `&quot;` it's easier to spot when reading the code.